src/TypeScript.UnitTests/Analyzer/EslintBridgeIssueConverterTests.cs

```diff
@@ -73,6 +73,58 @@ public void Convert_IssueConverted()
             convertedIssue.Severity.Should().Be(AnalysisIssueSeverity.Major);
         }
 
+        [TestMethod]
+        [DataRow("aaa", "aaa")]
+        [DataRow("aaa", "AAA")]
+        [DataRow("AAA", "aaa")]
+        [DataRow("AAA", "AAA")]
+        public void Convert_KeyMatchingIsCaseInsensitive(string keyInIssue, string keyInDefinition)
```

Comment:
New test for case-insensitive matching.

The previous code was inconsistent: the converter was case-sensitive when looking for the matching rule definition, but GetSonarRuleKey was case-insensitive.